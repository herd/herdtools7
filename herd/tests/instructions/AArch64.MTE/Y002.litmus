AArch64 Y002
(* IRG with exclusion *)
{
  int x=100;
  0:X0=x;
  0:X2=1;
  0:GCR_EL1=0x10000; (* RRND=1, Exclude=0 *)
}

P0           ;
IRG X1,X0,X2 ;

forall (not (0:X1=x:t0))

