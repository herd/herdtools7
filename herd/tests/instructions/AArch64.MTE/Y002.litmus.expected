Test Y002 Required
States 15
0:X1=x:t1;
0:X1=x:t10;
0:X1=x:t11;
0:X1=x:t12;
0:X1=x:t13;
0:X1=x:t14;
0:X1=x:t15;
0:X1=x:t2;
0:X1=x:t3;
0:X1=x:t4;
0:X1=x:t5;
0:X1=x:t6;
0:X1=x:t7;
0:X1=x:t8;
0:X1=x:t9;
Ok
Witnesses
Positive: 15 Negative: 0
Condition forall (not (0:X1=x:t0))
Observation Y002 Always 15 0
Hash=759815f2fc1b88479b10acaca049b937

